docs/publish-integrations/common-publish-integrations/get-approval-from-email.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Get Approval via Email
+description: Learn how to Get Approval via Email in Publish
+tags: [publish, email publish, approve]
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Get Approval via Email
+
+The **Get Approval via Email** step allows you to get approval from the email addresses entered as input in the step before moving on to the next steps in Publish.
+
+Based on your business requirements, you can designate certain email addresses to require approval, specify others as optional approvers, or set a minimum number of approvals needed from the provided email addresses.
+
+If some optional users reject, but there is still a chance to achieve the minimum approval count, the step will remain in `Waiting` status, awaiting responses from other users. For instance, if the minimum approval count is set to 3, and there are a total of 10 users with only one required user, even if 7 non-required users reject, there is still a possibility of obtaining approval from the remaining 3 users. However, if 8 non-required users reject, the step will fail, as the minimum requirement of 3 approvals cannot be fulfilled anymore.
+
+:::info
+
+Once a user makes a decision, it cannot be changed unless the step is restarted, even if the user sees the "Thank you" page. The step must be restarted to allow the user to make a new decision.
+
+When the step restarts or begins, all answers will be reset to `Waiting`, and users must provide their answers again. Consequently, they will receive approval emails again.
+
+:::
+
```

Comment:
@ftmzhrsckldr with last release, we changed the log structure of this step, maybe we can put it into here. I shared the example screen shot